tasks/scripts.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
+const getOptions = require('../lib/getOptions');
+const registerTaskWithProductionMode = require('../lib/registerTaskWithProductionMode');
+
 module.exports = (gulp, $, pkg) => {
+
   // @task: Build JS from components.
-  const task = async (args = {}) => {
+  function scriptTask(opts) {
```

Comment:
Why did you drop the async functionality?

Reply:
Why has it been added in the first place? No `await`s in the function, and there's no need to return a promise, is there?